variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+  default     = "ani-dev"
+}
+
+variable "location" {
+  description = "The Azure location where all resources in this deployment should be created"
+  default     = "uksouth"
+}
+
+variable "github_oauth_client_id" {
+  description = "Github oauth client id"
+}
+
+variable "github_oauth_client_secret" {
+  description = "Github oauth client secret"
```

Comment:
Similarly, some of these input values are sensitive, so [we should mark them](https://www.terraform.io/docs/language/values/variables.html#suppressing-values-in-cli-output)

Reply:
cool